packages/webapp/tailwind.config.js

```diff
@@ -30,6 +31,9 @@ module.exports = {
                 'dark-700': '#18181B',
                 'dark-800': '#09090B',
                 'bg-dark-blue': '#182633',
+                'state-green-900': '#84D65A',
+                'state-green-400': '#84D65A',
+                'row-hover': '#0d0d14',
```

Comment:
`row-hover` seems very generic

Reply:
yep, I lacked imagination happy to change if you have something. It's for the hover of table's row